__integrations__/__tests__/upsertMessageStatus.test.ts

```diff
@@ -0,0 +1,309 @@
+/* eslint-disable @typescript-eslint/no-use-before-define */
+/* eslint-disable sort-keys */
+import { exit } from "process";
+
+import { CosmosClient, Database } from "@azure/cosmos";
+import { createBlobService } from "azure-storage";
+
+import * as TE from "fp-ts/TaskEither";
+import * as E from "fp-ts/Either";
+import * as O from "fp-ts/Option";
+import { pipe } from "fp-ts/lib/function";
+
+import {
+  createCosmosDbAndCollections,
+  fillMessages,
+  fillMessagesStatus,
+  fillServices
+} from "../__mocks__/fixtures";
+
+import {
+  aFiscalCodeWithMessages,
+  aFiscalCodeWithoutMessages,
+  aMessage,
+  messagesList,
+  messageStatusList
+} from "../__mocks__/mock.messages";
+import { serviceList } from "../__mocks__/mock.services";
+import { createBlobs } from "../__mocks__/utils/azure_storage";
+import { getNodeFetch } from "../utils/fetch";
+import { upsertMessageStatus } from "../utils/client";
+import { log } from "../utils/logger";
+
+import {
+  WAIT_MS,
+  SHOW_LOGS,
+  COSMOSDB_URI,
+  COSMOSDB_KEY,
+  COSMOSDB_NAME,
+  QueueStorageConnection,
+  MESSAGE_CONTAINER_NAME
+} from "../env";
+import { NonEmptyString } from "@pagopa/ts-commons/lib/strings";
+import {
+  MessageStatus,
+  MessageStatusModel
+} from "@pagopa/io-functions-commons/dist/src/models/message_status";
+import { MessageStatusChange } from "@pagopa/io-functions-commons/dist/generated/definitions/MessageStatusChange";
+import { randomInt } from "crypto";
```

Comment:
This import causes fail on integration tests step.

Reply:
Thanks. Done in [b1f6e3c](https://github.com/pagopa/io-functions-app-messages/pull/10/commits/b1f6e3c9ff9ce32ae5d3cdcc7417238b04b46171)